pkgs/cupertino_http/example/ios/Runner/Assets.xcassets/LaunchImage.imageset/README.md

```diff
@@ -0,0 +1,5 @@
+# Launch Screen Assets
```

Comment:
Remove this file?

Remove this directory?

Reply:
This generated by the plugin template. Our other plugins include this as well e.g.
https://github.com/flutter/plugins/tree/main/packages/camera/camera_avfoundation/example/ios/Runner/Assets.xcassets/LaunchImage.imageset

Reply:
I'm not sure why we aren't cleaning up template cruft in other packages. I think we should always.

Reply:
> I'm not sure why we aren't cleaning up template cruft in other packages.

Just because cleaning cruft out of parts of example apps that probably nobody will ever look at is not a priority for us in the  ~90 {example x platform} sparse martrix we maintain in that repository alone. Remembering which parts of the templates—across six different platforms—would need to be adjusted, and how, every time we add or re-generate an example, would take a lot of reviewer energy that we would rather spend elsewhere.

> I think we should always.

PRs for the flutter/plugins+flutter/packages tooling that encode and enforce all the things that need to be adjusted would be welcome, since it would automate the process.